pkg/resource/promtail-client/promtail-client.go

```diff
@@ -154,8 +154,8 @@ func GetPromtailCredentials(lc loggedcluster.Interface, credentialsSecret *v1.Se
 	}
 
 	var credentials = make(map[string]string)
-	println(promtailYaml.Config.Clients)
-	println(promtailYaml.Config.Clients[0].URL)
+	println(promtailYaml)
```

Comment:
Why do we even need a println?

Reply:
It's a draft... not ready to review!
You're right, I'll do my debug test locally and not push them from now on :-)

Reply:
Ah my bad :) but we had println before 😂